src/native/libs/System.Globalization.Native/pal_collation.m

```diff
@@ -114,12 +114,15 @@ Range GlobalizationNative_IndexOfNative(const uint16_t* localeName, int32_t lNam
                                         const uint16_t* lpSource, int32_t cwSourceLength, int32_t comparisonOptions, int32_t fromBeginning)
 {
     assert(cwTargetLength >= 0);
-    Range result = {-2, 0};
+    Range result = {-1, 0};
```

Comment:
What do these magic numbers of -1, -2, -3, etc. mean?

Reply:
Added ErrorCodes enum to make it clear.